replicated-security/noble-1/noble-1-genesis-without-ccv.json

```
@@ -0,0 +1,233 @@
+{
+    "genesis_time": "2023-02-02T17:34:18.048979179Z",
```

```suggestion
    "genesis_time": "2023-02-09T15:00:00.000000000Z",
```
We have run into issues where the relayer couldn't be set up because the genesis time was too far in the past. It shouldn't be a big deal given the unbonding periods we have now, but just to give us ample time we can set the genesis time to one hour before spawn time.

---

good to know, i see no issue changing this.  Looks like it's 1 hour before spawntime now.